crates/cli/src/commands/install_all.rs

```diff
@@ -96,3 +107,39 @@ pub async fn install_all(session: ProtoSession) -> AppResult {
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::app::App;
+    use crate::session::ProtoSession;
+    use starbase::AppResult;
+
+    #[tokio::test]
+    async fn test_install_all_with_global() -> AppResult {
+        let app = App::new();
```

Comment:
@milesj I am not sure about how we initialize app and test proto.

Ahh, any suggestions you incude here? Are tests is in right direction? I think need to explore proto more( IMO will get experience more on it by contributing more to proto)

Reply:
@varshith257 We use integration tests for these, here's some refs: https://github.com/moonrepo/proto/blob/master/crates/cli/tests/use_test.rs